phpBB/language/en/cli.php

```diff
@@ -110,8 +110,10 @@
 
 	'CLI_EXTENSION_DISABLE_FAILURE'		=> 'Could not disable extension %s',
 	'CLI_EXTENSION_DISABLE_SUCCESS'		=> 'Successfully disabled extension %s',
+	'CLI_EXTENSION_ALREADY_DISABLED'	=> 'Extension %s was already disabled',
```

Comment:
No, that is correct as I see it (but I would prefer present rather than past, but that is just taste).
For me, "Extension %s is already enabled" is ok.